code/modules/codex/entries/faction_codex.dm

```diff
@@ -0,0 +1,88 @@
+    --Navy is spread out thinly."
+    
+/datum/codex_entry/NT_faction
+    display_name = "NanoTrasen (faction)"
+    lore_text = "NanoTrasen, known informally as NT, is the largest corporation in human space, with facilities mainly in TG space along with a few \
+    frontier outposts. They focus on phoron research and mining.<br> NanoTrasen was started in in 2057, their main invention back then being a phoron-engine design. \
+    Ever since, NT has gained the favor of the ISA and TG, allowing them to rapidly expand operations, eventually encompasssing most human space. \
+    Despite their generally good PR and wide acclaim, vocal critics of NT indicate their abuses of workers, as well as their controversial production of \
+    vatborns and synthetics for the military. Despite this, NT is projected to keep growing, even with the TG-ICC war.<br> \
+    mechanics_text = "Several million employees.<br><br>Has a few thousand armed security officers.<br><br>~100 FTL ships, mainly non-combatant or escorts<br><br>\
+    ++Great scientific capabilities.<br>\
+    +Posseses a large amount of influence.<br>\
+    +Makes large amounts of revenue.<br>\
+    --More vocal criticism in recent years."
+
+/datum/codex_entry/ICC_faction
+    display_name = "Independent Colonial Confederation (faction)"
+    lore_text = "The Independent Colonial Confederation, known informally as the ICC, is an alliance of colonies, habitats and space stations, \
+    which revolted against the Terran Government due to a variety of ideological differences.<br> The Confederation's three main planets are Gilgamesh, \
+    known as the diplomatic and governmental center and formed by the two habitat stations that fled Earth orbit, Caelus, formed by Earth nationalists, \
+    a group of free thinkers and middle-class men united in their beliefs against the Terran Government and its authoritarian rule, and Cydonia, \
+    inhabited by descendants of the Sons of Mars, known as the naval, military and engineering powerhouse of the ICC due to its large expertise.<br> \
+    mechanics_text = "~110 million people.<br><br>Standing army of 500.000. (SoM troops included)<br><br>275 FTL ships, largely formed of converted ships (SoM ships included)<br><br>\
+    ++High military and civilian morale.<br>\
+    +Good engineering capabilites.<br>\
+    +Good defensive/guerilla tactics.<br>\
+    -Largely unable to go on the offensive.<br>\
+    -Ideological and military differences between the SoM and rest of the ICC.<br>\
+    --Very small population and military compared to the Terran Government."
+    
+/datum/codex_entry/SoM_faction
+    display_name = "Sons of Mars (faction)"
+    lore_text = "The Sons of Mars, known informally as the SoM, is an organized government with major holdings in the ICC space, \
+    tracing its roots back being to a crime syndicate on Mars.<br> The Sons of Mars center of operations is on Cydonia, a planet inhabited by , \
```

Comment:
> The Sons of Mars, known informally as the SoM, is an organized government with major holdings in the ICC space, \
    tracing its roots back being to a crime syndicate on Mars

Sons of Mars are an organized rebellion formed as a result of TG's authoritarian treatment of the Martian people, not a crime syndicate.